manual/opt.tex

```diff
@@ -153,9 +153,110 @@ \subsubsection{quartic}
+   &   \texttt{nkept} &  integer   & $>0$ &  & \# of eigenvectors to keep per block in BLM\\
+  \hline
+\end{tabularx}
+\end{center}
+\end{table}
+
+Additional information:
+\begin{itemize}
+  \item \texttt{shift\_i}.  This is the initial value of direct term added to the diagonal of the Hamiltonian matrix. 
+                            More stable but slower optimization with a large value. The used value is auto-adjusted by the optimizer. 
+  \item \texttt{shift\_s}.  This is the initial value of the stabilizer based on the overlap matrix added to the Hamiltonian matrix. 
+                            More stable but slower optimization with a large value. The used value is auto-adjusted by the optimizer. 
+  \item \texttt{omega}.     Since approximate wave function has a finite variance of energy, is has been proved that the $\omega$ value
+                            needs to be ``shifted down'' by the amount of the standard deviation of the energy to target the correct state. 
+  \item \texttt{nblocks}.   This is the number of blocks used in block LM. The amount of memory required to store LM matrices decreases
+                            with increased number of blocks. But the error introduced by BLM would increase with number of blocks.  
+  \item \texttt{nolds}.     In BLM, the inter-block correlation is accounted for by including a small number of wave function update vectors
+                            outside the block. Larger \texttt{nolds} would include more inter-block correlation and more accurate results, but 
+                            also higher memory requirements. 
+  \item \texttt{nkept}.     This is the number of update directions taken from each block in BLM. If all directions are taken in each block, 
+                            BLM becomes standard LM and no approximation is made. Normally taking less than 5 directions in each block would 
+                            be sufficient. 
+                            
+\end{itemize}
+
+
+Recommendations:
+\begin{itemize}
+  \item Default shift\_i, shift\_s should be fine. 
+  \item In order to obtain an unbiased excitation energy, one should optimize the ground state with the excited state variational principle as well by setting
+        \texttt{omega} below the ground state energy.  Note that using the ground state variational principle for the ground state and the excited state variational
+        principle for the excited state creates a bias in favor of the ground state. 
+  \item Setting $\omega$ correctly is very important since a too low or too high $\omega$ will lead to a failure of the optimization. We recommend that one use
+        $\omega=E-\sigma$, in which $E$ is the desired excited state's energy and $\sigma^2$ is your approximate wave function's energy variance.
+  \item The BLM has a higher stochastic prefactor than the standard LM, and so we only advise its use when at least 5,000 variables are being optimized.
```

Comment:
I assume the stochastic prefactor refers to the run time?

Would the influence on run time be clearer (if not as technically correct) as something like:
The BLM has higher runtime overhead than the standard LM, but better scaling in the number of variables, so we only advise ...